packages/replay/jest.setup.ts

```diff
@@ -15,27 +15,6 @@ jest.mock('./src/util/isBrowser', () => {
   };
 });
 
-afterEach(() => {
```

Comment:
This seems to not be necessary anymore (and actually lead to test issues), so I removed it.